node/bft/src/helpers/pending.rs

```diff
@@ -23,19 +23,20 @@ use std::{
 use time::OffsetDateTime;
 use tokio::sync::oneshot;
 
-#[cfg(not(test))]
-pub const NUM_REDUNDANT_REQUESTS: usize = 2;
-#[cfg(test)]
-pub const NUM_REDUNDANT_REQUESTS: usize = 10;
-
 const CALLBACK_TIMEOUT_IN_SECS: i64 = MAX_FETCH_TIMEOUT_IN_MS as i64 / 1000;
 
+/// Returns the maximum number of redundant requests for the specified number of validators.
+pub const fn max_redundant_requests(num_validators: usize) -> usize {
+    num_validators.saturating_div(2)
```

Comment:
```suggestion
    1 + num_validators.saturating_div(3)
```

It is adequate to set this value to the **availability threshold**, as with high probability one will respond honestly (in the best case and worst case with stake spread across the validators evenly and unevenly, respectively).

Thus, the proposed change is to set it to `1 + num_validators.saturating_div(3)`.